content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -1,29 +1,26 @@
 ---
-title: Role-Based Access Control
+title: Authorization Policy
 description: Shows how to set up role-based access control for services in Istio mesh.
 weight: 40
-keywords: [security,access-control,rbac]
+keywords: [security,access-control,rbac,authorization]
 ---
 
-This task shows how to set up role-based access control (RBAC) for services in Istio mesh. You can read more about Istio
-RBAC from [Istio RBAC concept page](/docs/concepts/security/#role-based-access-control-rbac).
+This guide covers the main tasks you might need to perform to set up Istio authorization, also known
+as Istio Role Based Access Control (RBAC), for services in an Istio mesh. You can read more in
+[Istio authorization](/docs/concepts/security/#role-based-access-control-rbac).
```

Comment:
"and get started with a basic tutorial in Istio Security Basics."

Reply:
Done, it would be better to link to the page, do you mean this page (https://preliminary.istio.io/docs/tasks/security/authn-policy/) or another page?

Reply:
There will be a security CUJ page, which we can link to later.

Reply:
Great!